src/traits/to_impl.rs

```diff
@@ -0,0 +1,5 @@
+use super::*;
+
+pub unsafe trait ToImpl<T: Interface> {
```

Comment:
Can we document this? (especially what is needed to use this safely - even if it's not meant to be used directly by end users)

Reply:
Added docs.